### ISSUE TITLE
[Issue] regctl image inspect: documented format options for reading labels don't work after upgrading to v0.5.2

### ISSUE THREAD

**Issue by jokimaki** [regclient/regclient#566]
I've used examples from https://github.com/regclient/regclient/blob/main/docs/regctl.md#format-flag to inspect image labels. After upgrading to v0.5.2 I get an error `can't evaluate field Config in type *blob.BOCIConfig`.

v0.5.1 works:
```
docker container run -i --rm --net host \
  -v regctl-conf:/home/my-username/.regctl/ \
  ghcr.io/regclient/regctl:v0.5.1 \
  image inspect --format '{{index .Config.Labels "org.opencontainers.image.version"}}' regclient/regctl:latest

v0.5.2
```

v0.5.2 produces an error:
```
docker container run -i --rm --net host \
  -v regctl-conf:/home/my-username/.regctl/ \
  ghcr.io/regclient/regctl:v0.5.2 \
  image inspect --format '{{index .Config.Labels "org.opencontainers.image.version"}}' regclient/regctl:latest

template: out:1:15: executing "out" at <.Config.Labels>: can't evaluate field Config in type *blob.BOCIConfig
```

Is this a regression in v0.5.2 or has the syntax changed and the documentation is out-of-date?

**Comment by sudo-bmitch** [comment]
It's a regression, fixed in #554. I'll get a new release out this weekend since it looks like it's affecting multiple people.

**Comment by jokimaki** [comment]
Sounds great, thank you! Totally missed that closed issue, sorry for the duplicate.

**Comment by sudo-bmitch** [comment]
Thanks for the report, and no worries on the duplicate. This is fixed in v0.5.3 that was just released. If it gives you any trouble, don't hesitate to reach out.

https://github.com/regclient/regclient/releases/tag/v0.5.3
